roles/kube-cni/tasks/main.yml

```diff
@@ -12,14 +12,21 @@
   delay: 3
   ignore_errors: yes
 
-# ----------- flannel
-- name: Configure bridge cni plugin 
+# ----------- bridge
+- name: Configure bridge cni plugin
   shell: >
     kubectl create -f /etc/bridge.yaml
   args:
     creates: "{{ kubectl_home }}/.kubeadm-podnetwork-complete"
   when: pod_network_type == "bridge"
 
+# ----------- kokonet-bridge
+- name: Configure bridge cni plugin
+  shell: >
```

Comment:
I think you can use `command` here instead of `shell` (which will allow it to pass for `ansible-lint`). I know the other commands use `shell` but they should eventually be changed too.

Reply:
How about to address the changes in one shot?
I've added issue ticket.